R/getSymbols.R

```diff
@@ -631,7 +631,13 @@ function(Symbols,env,return.class='xts',
             if(verbose) {
                 cat(paste('Loading ',Symbols[[i]],paste(rep('.',10-nchar(Symbols[[i]])),collapse=''),sep=''))
             }
-            query <- paste("SELECT ",paste(db.fields,collapse=',')," FROM ",Symbols[[i]]," ORDER BY date")
+          
+            if (!is.null(from) & !is.null(to)) {
```

Comment:
This implies that the user must specify *both* `from` and `to` (i.e. only specifying one will cause all data to be returned). It would be better if they could specify none, one, or both.